picard/config_upgrade.py

```diff
@@ -246,6 +241,27 @@ def upgrade_to_v2_0_0_dev_3():
             _s[opt] = _CAA_SIZE_COMPAT[value]
 
 
+def upgrade_to_v2_1_0_dev_1():
+    """Upgrade genre related options"""
+    print("upgrade_to_v2_1_0_dev_1")
+    _s = config.setting
+    if _s["folksonomy_tags"]:
```

Comment:
Check if `"folksonomy_tags" in _s` ?

Reply:
Is this needed? We access config.setting['folksonomy_tags'] elsewhere without test. But I could also do a `if _s.get('folksonomy_tags')`

Reply:
I replaced it with `_s.get('folksonomy_tags')`